### ISSUE TITLE
mate-mint-menu not working in a fresh Mate install

### ISSUE THREAD

**Issue by karasu** [Antergos/antergos-packages#57]
mate-mint-menu not working in a fresh Mate install

@lots0logs have you checked this? I just tried to install Mate in a VM and this menu is not working

**Comment by lots0logs** [comment]
hmmm..I will check it asap!

**Comment by j883376** [comment]
It looks like the latest build that depends on the new python2-no-hardening package fixed this but created a dependency issue with certain packages that depend on python2.

When trying to install python2-pyopenssl, it requires python2>=2.7 but the new package is not solving the dependency correctly.

$ sudo pacman -S python2-pyopenssl 
resolving dependencies...
looking for conflicting packages...
:: python2 and python2-no-hardening are in conflict. Remove python2-no-hardening? [y/N] y
error: failed to prepare transaction (could not satisfy dependencies)
:: mate-mint-menu: requires python2-no-hardening

**Comment by lots0logs** [comment]
@j883376 I just pushed an update to the package. Please try it and let me know if it resolves the problem. Thanks.

**Comment by j883376** [comment]
The update seems to have resolved this. I was able to install python2-pyopenssl and mate-mint-menu is working fine.

**Comment by lots0logs** [comment]
Ok, great. Thanks for the confirmation!

**Comment by j883376** [comment]
On a different computer I'm getting segfaults when running any Python program with python2-no-hardening installed. I've had to uninstall this and go back to the default Arch package.

The original crashing issue with mate-mint-menu was fixed upstream here:

https://github.com/linuxmint/mintmenu/pull/140

Can the package be updated to use the latest source and change the dependency back to the Arch python2 package?
